css/style.css

```diff
@@ -0,0 +1,285 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  background-color: #f4f5f6;
+}
+
+.container-fluid {
+  min-width:  100%;
+  padding:0 !important;
+}
```

Comment:
- [ ] Try not to use `!important` if it can be avoided. This can be done with the bootstrap class `p-0`

Reply:
Thank you for your feedback, We worked on the corrections and we are still waiting for another code review and hopefully approval.